crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -31,6 +31,11 @@ use super::error::BlockchainError;
 /// The value of a single coin in satoshis.
 pub const COIN_VALUE: u64 = 100_000_000;
 
+/// The version tag to be prepended to the leafhash. It's just the sha512 hash of the string
+/// `UtreexoV1` represented as a vector of [u8] ([85 116 114 101 101 120 111 86 49]).
+/// The same tag is "5574726565786f5631" as a hex string.
+pub static UTREEXO_TAG_V1: &[u8] = "5b832db8ca26c25be1c542d6cceddda8c145615cff5c35727fb3462610807e20ae534dc3f64299199931772e03787d18156eb3151e0ed1b3098bdc8445861885".as_bytes();
```

Comment:
In this case, `as_bytes` will encode each `char` as it's ascii correspondent, not as a byte sequence, so this evaluates to:
```
[53, 98, 56, 51, 50, 100, 98, 56, 99, 97, 50, 54, 99, 50, 53, 98, 101, 49, 99, 53, 52, 50, 100, 54, 99, 99, 101, 100, 100, 100, 97, 56, 99, 49, 52, 53, 54, 49, 53, 99, 102, 102, 53, 99, 51, 53, 55, 50, 55, 102, 98, 51, 52, 54, 50, 54, 49, 48, 56, 48, 55, 101, 50, 48, 97, 101, 53, 51, 52, 100, 99, 51, 102, 54, 52, 50, 57, 57, 49, 57, 57, 57, 51, 49, 55, 55, 50, 101, 48, 51, 55, 56, 55, 100, 49, 56, 49, 53, 54, 101, 98, 51, 49, 53, 49, 101, 48, 101, 100, 49, 98, 51, 48, 57, 56, 98, 100, 99, 56, 52, 52, 53, 56, 54, 49, 56, 56, 53]
```
I think what you're looking for is:

```rust
pub const UTREEXO_TAG_V1: [u8; 64] = [
    0x5b, 0x83, 0x2d, 0xb8, 0xca, 0x26, 0xc2, 0x5b, 0xe1, 0xc5, 0x42, 0xd6, 0xcc, 0xed, 0xdd, 0xa8,
    0xc1, 0x45, 0x61, 0x5c, 0xff, 0x5c, 0x35, 0x72, 0x7f, 0xb3, 0x46, 0x26, 0x10, 0x80, 0x7e, 0x20,
    0xae, 0x53, 0x4d, 0xc3, 0xf6, 0x42, 0x99, 0x19, 0x99, 0x31, 0x77, 0x2e, 0x03, 0x78, 0x7d, 0x18,
    0x15, 0x6e, 0xb3, 0x15, 0x1e, 0x0e, 0xd1, 0xb3, 0x09, 0x8b, 0xdc, 0x84, 0x45, 0x86, 0x18, 0x85,
];
``

Reply:
Thanks for the review! Applied in the latest push